src/transformers/pipelines/__init__.py

```diff
@@ -505,7 +505,7 @@ def clean_custom_task(task_info):
 
 def pipeline(
     task: str = None,
-    model: Optional = None,
+    model: Optional[Union[str, Path]] = None,
```

Comment:
I think the instances are `str`, `PreTrainedModel` or `TFPreTrainedModel`

Reply:
thanks for the fast reply. So Path object shouldn't be there?